Polyhedron/demo/Polyhedron/Plugins/Mesh_3/C3t3_io_plugin.cpp

```diff
@@ -499,6 +499,9 @@ try_load_a_cdt_3(std::istream& is, C3t3& c3t3)
   }
   std::getline(is, s);
   if(s != "") {
+    if(s[s.size()-1] == '\r') { // deal with Windows EOL
+      s.resize(s.size() - 1);
+    }
     if(s != std::string(" ") + CGAL::Get_io_signature<Fake_CDT_3>()()) {
```

Comment:
Note that there exist those convenient functions:
  - `std::string::back()`. and
  - `std::string::pop_back()`.

I also wonder why that is required. What is the scenario that can lead to a `\r` in the file?

Reply:
If the user is running on windows and saves the c3t3 without setting `std::ios_base::binary`, then the end of line will be `\r\n` :
```cpp
std::ofstream output_file("out.cgal");
CGAL::IO::save_binary_file(output_file, c3t3, /*binary=*/false);
output_file.close();
```
But, if the user saves with the setting `std::ios_base::binary` enabled, then the end of line will be `\n` :
```cpp
std::ofstream output_file("out.cgal", std::ios_base::binary);
CGAL::IO::save_binary_file(output_file, c3t3, /*binary=*/false);
output_file.close();
```

Reply:
But the first instance should not happen, `save_binary_file` should always be called on a file with `std::ios_base::binary` enabled. We should probably add an assertion in this method.
Also, `load_binary_file` checks for `\r`, and his code have been duplicated in `C3t3_io_plugin.cpp`. if the check is not needed, we should probably also delete it here.

Reply:
It looks like a bug from the user, to use that function using an `std::ostream` not in binary mode. Probably that should be checked, and make the function fail (with `failbit`), or using an assertion.